wiki/en/Running-a-Server.md

```diff
@@ -36,86 +37,108 @@ However, various problems can also arise when setting up Servers - especially wh
 
 Consider using a cloud host, not your home internet connection, to get better ping times if you're having problems.
 
-
 ### General notes
 
 * Any Server should have at least 1.6GHz CPU frequency and 1GB RAM
 * Running a Server may require you to adjust any firewalls running on or outside of your machine or cloud host.
-* Running an Unregistered Server at home will require you to [port forward](#port-forwarding) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
-* Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the Client and Server. (There are plans to expand IPv6 support.)
+* You must set up port forwarding on your router to run an [Unregistered Server](Unregistered-Servers) at home. This should not be necessary when running a Registered Server in most cases. However, some home networks can require port forwarding for a Registered Server.
+* Jamulus offers limited IPv6 support that you can turn on for a Client or Server from the command line.
 
-## Server Types
+---
 
-You can run your Server in different ways (either at home or on a 3rd party host):
+## Installation
 
-### 1. Registered
+Most people run Jamulus on a 3rd party/cloud host as a "headless" Server (no video display or keyboard) on **hardware without audio** running Linux. You can also run a Server in a [**desktop environment**](#servers-on-the-desktop).
 
-Your Server will be listed in a Directory. By default, Jamulus has a list of built-in Directories that Clients can connect to. If you register with one of these, anyone can then discover and connect to your Server. You can also have your Server listed on a Custom Directory, if that better meets your needs.
+To run a headless server on Linux, the following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution that uses systemd. 
 
-<figure>
-	<img src="{% include img/en-screenshots/diagram-reg-server.inc %}" loading="lazy" alt="Diagram of connections between Clients within a Jamulus Registered Server">
-<figcaption>How Registered Servers work</figcaption>
-</figure>
+If you want to run a Server on a **Raspberry Pi** (or a different armhf/arm64 debian-based device), you will need to download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }}) depending on your CPU - not the default `amd64` ones for use on Intel/AMD machines.
```

Comment:
I downloaded arm64 for the first time yesterday, to use on an Ubuntu 22 platform. It seems like Ubuntu 22 can run as arm64 or amd64. I admit that I still don't know how to render this PR to review the built site. Does this sound like only Raspi would use arm64? I suspect we should use a table with three entries to explain the three platforms, rather than inlining for these links.

Reply:
The majority of people looking at this page would just want the standard amd64. The alternative download links are presented for the minority who will know what they need. The same principle of not confusing/annoying the majority for the sake of the minority applies on the macOS and Windows download pages today (for legacy and JACK versions respectively).

Reply:
I keep looking at the list of publicly listed servers. Most are Linux. How do you know most of those are AMD64?